config/locales/en.yml

```diff
@@ -80,6 +80,8 @@ en:
     fine_payment:
       accept_html: <span class="font-weight-bold">Success!</span> $%{amount} paid. A receipt has been sent to the email address associated with your account. Payment may take up to 5 minutes to appear in your payment history.
       cancel_html: <span class="font-weight-bold">Payment canceled.</span> No payment was made &mdash; your payable balance remains unchanged.
+      request_failed_html: <span class="font-weight-bold">Sorry!</span> Something went wrong, possibly due to a connection failure. Please contact us for help resolving your fines.
+      payment_failed_html: <span class="font-weight-bold">Sorry!</span> Payment failed. Please contact us for help resolving your fines.
```

Comment:
Should there be directions for contacting us?

Reply:
Yeah, I should make that a link, probably